cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -99,10 +124,20 @@ func Run(gadgetRoot, device string, options *Options) error {
 		}
 	}
 
+	// store the encryption key as the last part of the process to reduce the
+	// possiblity of exiting with an error after the TPM provisioning
+	if options.Encrypt {
+		if err := key.Store(options.KeyFile); err != nil {
```

Comment:
This is where the sealed key data file will be stored eventually, isn't it?

Reply:
Yes. Currently it's only storing the plaintext file.